#231915 Update Notification banner to use GDS

PauliePension opened the pull request:
Deleted Notification banner scss, and changed the included  button to default style (not reveresed) in the demo app.

PauliePension committed:
#231915 Deleted Notification banner scss, and changed the button to  default style (not reveresed) in the demo app.

sussexrick commented on ThePensionsRegulator.Frontend/Styles/_tpr-notification-banner.scss:
Deleting this file entirely leaves some pretty strange alignment on the buttons, and unwanted extra bottom margin:

<img width="969" height="433" alt="Dismiss button neither left aligned or centred" src="https://github.com/user-attachments/assets/0ccc6e67-04f9-4d47-bef7-75478f177c5d" />

I've done some experimenting and found that this alignment is best fixed with CSS to neutralise the effect of a `.govuk-width-container` class rather than trying to remove the `.govuk-width-container`. So `_tpr-notification-banner.scss` should remain with the following content:

```scss
.govuk-notification-banner {

    .govuk-notification-banner__content {
        > .govuk-width-container {
            margin: 0;

            > .govuk-grid-row {

                .govuk-button {
                    margin-bottom: 0;
                }

                .govuk-button-group {
                    margin-bottom: 0;
                }
            }
        }
    }
}
```

PauliePension replied:
done

sussexrick commented on ThePensionsRegulator.Frontend/Styles/_tpr-notification-banner.scss:
The backoffice preview also needs a little love.

<img width="788" height="371" alt="Backoffice preview" src="https://github.com/user-attachments/assets/13e5df2f-5de2-4db7-ba29-9502329bc296" />


I think it needs:

* to correct the spacing in the header
* to be green or blue depending on the selected option
* to have some spacing after the "1 block" block

PauliePension replied:
done

PauliePension committed:
Merge from develop

PauliePension committed:
Updaintf notification-banner changes

PauliePension committed:
PR Fix
Removed the 'backoffice-block-view' class. It's spacing requirements interfers with the bottom  padding on the content.
Added correct class depending on which 'type' you select.
Changed correct title to display depnding on Type and title.

PauliePension committed:
Merge branch 'develop' into story/231915-Update-the-start-page-to-use-correct-notification-banner-component

PauliePension committed:
bump version

sussexrick reviewed (commented):
Happy to approve but you've now got a conflict on the version because later versions have been released in the meantime.

I'd like to hold off on merge for now though, until we have confirmation from the head of design that she's ready for us to push this change out. I've chased.

PauliePension committed:
Merge with develop

PauliePension committed:
#231915 updated build props minor id
